packages/enzyme/src/ReactWrapper.js

```diff
@@ -728,7 +727,8 @@ class ReactWrapper {
    * @returns {String}
    */
   name() {
-    return this.single('name', n => displayNameOfNode(n));
+    const adapter = getAdapter(this[OPTIONS]);
+    return this.single('name', n => adapter.displayNameOfNode(n));
```

Comment:
i think this needs `adapter.displayNameOfNode ? adapter.displayNameOfNode(n) : displayNameOfNode(n)`